kbase-extension/static/kbase/js/widgets/narrative_core/kbaseCellToolbarMenu.js

```diff
@@ -418,10 +418,7 @@ define([
                                 id: events.addEvent({type: 'click', handler: doToggleMinMaxCell})
                             }, [
                                 span({
-                                    class: 'fa fa-' + toggleIcon + '-square-o fa-lg',
-                                    style: {
-                                        color: color
-                                    }
+                                    class: 'fa fa-' + toggleIcon + '-square-o fa-lg'
```

Comment:
`fa fa-${toggleIcon}-square-o fa-lg`?

Reply:
This was already in place, not changed by the PR. The change was to remove the explicit color style.
But maybe I don't know what the question is?
The `toggleIcon` in this case is the functional part of the font awesome class name, which in this case is either fa-minus-square-o or fa-plus-square-o , so basically toggles between the minus and plus icons.
I was tempted to, but did not, improve that line by making it a template string.

Reply:
The comment was just about using a template string. ATM we don't have any policies either way so it's up to you. 🙂

Reply:
Oh, right, well, I have tried to restrain myself from just changing code to be better. I do try to restrain myself to only changing code if
(a) it is required for the fix
(b) it is a latent bug not described in the bug report
(c) it is seriously required to have to code make sense (a big exception is any jQuery code, which never makes sense.)
(d) it is required for a test or makes a test easier or more reliable, e.g. an attribute to select on
(e) it angers eslint
I'm sure there are others ...

That said, I do sometimes just make code improvements if they bug me enough and I'm in a generous mood.
In this case, string concatenation is old fashioned, but there are literally hundreds of not thousands of locations in the codebase where they are used and can be transformed into template strings :)
It sure is satisfying to do it, though :)

Reply:
(c) - :grin:

Template strings can be implemented globally by `eslint`, should we wish to adopt a policy towards them at some point.